src/components/FeatureViews/Registration/RegistrationCourseEnrollments.js

```diff
@@ -32,261 +41,349 @@ import gql from "graphql-tag";
-                                {/*                    </span>*/}
-                                {/*                    <span className="expanded-text">*/}
-                                {/*                        <b>School Teacher</b>:{" "}*/}
-                                {/*                        {notes["Current Instructor in School"]}*/}
-                                {/*                        <br />*/}
-                                {/*                    </span>*/}
-                                {/*                    <span className="expanded-text">*/}
-                                {/*                        <b>Textbook:</b> {notes["Textbook Used"]}*/}
-                                {/*                        <br />*/}
-                                {/*                    </span>*/}
-                                {/*                </Typography>*/}
-                                {/*            </Paper>*/}
-                                {/*        </TableCell>*/}
-                                {/*    </TableRow>*/}
-                                {/*)}*/}
-                            </Fragment>
-                        );
-                    })}
-                </TableBody>
-            </Table>
-            <Dialog aria-describedby="unenroll-dialog-description"
-                aria-labelledby="unenroll-dialog-title"
-                className="session-view-modal"
-                fullWidth
-                maxWidth="xs"
-                onClose={closeUnenrollDialog(false)}
-                open={unenroll.open}>
-                <DialogTitle id="unenroll-dialog-title">
-                    Unenroll in {courseTitle}
-                </DialogTitle>
-                <Divider />
-                <DialogContent>
-                    <DialogContentText>
-                        You are about to unenroll in <b>{courseTitle}</b> for{" "}
-                        <b>
-                            {unenroll.enrollment && fullName(enrollments.find(({ id }) => id == unenroll.enrollment).student.user)}
-                        </b>
-                        . Performing this action will credit the remaining enrollment
-                        balance back to the parent's account balance. Are you sure you want
-                        to unenroll?
-                    </DialogContentText>
-                </DialogContent>
-                <DialogActions>
-                    <Button
-                        variant="outlined"
-                        color="secondary"
-                        onClick={closeUnenrollDialog(true)}
-                    >
-                        Yes, unenroll
-                    </Button>
-                    <Button
-                        variant="outlined"
-                        color="primary"
-                        onClick={closeUnenrollDialog(false)}
+                  </AccordionSummary>
+
+                  <AccordionDetails className={classes.accordionNotesBorder}>
```

Comment:
let's move the `<AccordionDetails/>` component into its own component. pass in any necessary data through props